Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx

```diff
@@ -275,8 +289,19 @@ export const editorSteps: { [key in ManifestEditorSteps]: EditorStep } = {
     content: SelectDialogs,
     editJson: false,
     subText: () =>
-      formatMessage(
-        'The capabilities of your bot are defined in its dialogs and triggers. Selected dialogs will be included in the manifest. Internal dialogs or actions may not be relevant to other bots. Learn more.'
```

Comment:
Unless this has changed, you should be able to set the `helpLink` property, and it should take care of the link for you. If it has changed, please update the type to reflect that. 

https://github.com/microsoft/BotFramework-Composer/blob/1990c8076249aeb49209a1b7c99794c258edf17e/Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx#L122

Reply:
Use `helpLink`